### DISCUSSION TITLE
[灌水] 饮茶时间

### DISCUSSION THREAD

**Discussion by ZhuYuan-148** [share121/inter-knot#2518] [其他]
![mmexport1723875235031](https://github.com/user-attachments/assets/8cf6b7e9-c46d-4360-9113-86d1ad3c6334)
难得的闲暇时间，前辈把大家聚在一起品茶，这股松弛感真是看不出一点治安局的样子呢…
![mmexport1723875236173](https://github.com/user-attachments/assets/0d274e8b-8b2b-4485-b930-8c0d0a7927b1)

> 美图作者：pixiv HAKUMA

**Comment by LinkboyTvT** [comment]
绳匠网上冲浪中

**Comment by nhb-bcd** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/f7763e07de9cad339ce6adc3c8f5e807.png" alt="煮饭" width="75" >
难得休闲

**Comment by Obsidia-cdl** [comment]
诶..各位治安局的前辈们好啊，刚上岗不久，原来前辈们都喜欢泡茶啊

**Reply by ZhuYuan-148** [reply]
也不是都喜欢…准确来说只有青衣前辈喜欢

**Comment by wwwsheepwww** [comment]
好图！
